src/components/ListCard.js

```diff
@@ -31,16 +32,19 @@ const timeToString = time => {
 };
 
 const ListCard = props => {
-  const { classes, data } = props;
+  const { classes, report } = props;
   return <Card className="card">
     <CardContent className={classes.allContent}>
       <CardMedia className="cardPicture"
-                 image={data.image_url ? data.image_url : DEFAULT_IMAGE_URL}
+                 image={report.data.mediaPaths!==undefined && report.data.mediaPaths.length>0 ? report.data.mediaPaths[0] : DEFAULT_IMAGE_URL}
```

Comment:
tiny nit: could we do `report.data.mediaPaths && report.data... ? ...`? That way if it ends up some other falsy value (null, false, etc) we still don't display it.